src/storage/src/healthcheck.rs

```diff
@@ -260,14 +292,16 @@ impl OverallStatus {
             OverallStatus::Starting => "starting",
             OverallStatus::Running => "running",
             OverallStatus::Stalled { .. } => "stalled",
+            OverallStatus::PermanentlyStalled { .. } => "permanently stalled",
```

Comment:
The original design included a `halted` state which seems like it covers this case: https://www.notion.so/materialize/Exposing-source-sink-errors-via-SQL-1c83b2070b5a48db967b829b7ccd88dc?pvs=4#0f93175a1fe84133be7e1b531c64e938

Does that sound right? We don't have to use the original naming, but it might be worth mentioning somewhere why we're changing it.

Reply:
changed it to `failed`!